src/Middleware/AuthMiddleware.php

```
@@ -25,6 +25,12 @@ class AuthMiddleware
     public function handle(Auth $auth, Response $response): bool
     {
         if (!$auth->check()) {
+            if ($this->isAjaxRequest()) {
+                Response::applySecurityHeaders('app');
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\StratFlow\Core\Response' in method 'handle'. (undefined)

(StaticAccess)